### ISSUE TITLE
Possibility of running on 3rd party board

### ISSUE THREAD

**Issue by jimmy-adams** [doonny/PipeCNN#172]
Dear all,
I want to ask if it is possible to run it on the 3rd party board with the same zynq ultrascale+ fpga core.
due to my limited experience, i need to generate the wrapper file with the zynq processing system, and then put that into the vitis as the platform.
I don't know what should i configure in Vivado for the bitstream generating. Can you explain with your knowledge?

Thanks in advance

**Comment by doonny** [comment]
Yes, you could run pipecnn on 3rd party board.   First thing you need is to develop a custom platform for mpsoc. Then, everything is the same.
